views/index.erb

```diff
@@ -0,0 +1,5 @@
+<form action= '/play' method = 'post'>
+<label for="player_1_name">Enter player 1's name:</label><br>
```

Comment:
This label makes it really clear in the browser what you wish the user to enter.